Algorithms/GEMETD.py

```diff
@@ -0,0 +1,56 @@
+from Algorithms.ETDLB import ETDLB
+from Algorithms.BaseTD import BaseTD
+import numpy as np
+
+
+class GEMETD(ETDLB):
+    """
+    An ETD(0) implementation that uses GEM (aka GTD2(0) with x and x_p switched) to estimate emphasis.
+    """
+    def __init__(self, task, **kwargs):
+        super().__init__(task, **kwargs)
+        self.beta = self.task.GAMMA
+        self.gem_alpha = self.get('gem_alpha')  # Step size for GEM weights.
+        self.gem_eta = self.get('gem_eta')  # Regularization parameter for GEM; not needed for a fixed target policy.
+        self.k = np.zeros(self.task.num_features)  # Auxiliary weights for GTD2.
```

Comment:
We might want to change ``self.k`` to ``self.v`` to be consistent with other Gradient-TD algorithms? What do you think?

Reply:
I was thinking `v` and `w` should be reserved for the weight vectors related to the value function. GEM is using two new weight vectors to learn an estimate of the emphasis for the current feature vector, so `self.k` is fulfilling a different role than `self.v`.

Reply:
Got it.